vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -343,6 +344,8 @@ void select(GridTRElement row) {
         if (isMultiselect()) {
             executeScript("arguments[0].selectItem(arguments[1]._item);", this,
                     row);
+            executeScript("arguments[0].$server.select(arguments[1]._item.key)",
```

Comment:
I'm kind of wondering about the utility of this class if it just uses some internal API that calls the server directly. Maybe we should actually click the checkboxes, as was done in the test case.
I'm going to take a look at this.

Reply:
> we should actually click the checkboxes, as was done in the test case.

I was implementing it that way first as you see from the history, but then I noticed that the method needs to work also in single select mode, when the checkbox is not there.

Reply:
Attempted an alternative fix here: https://github.com/vaadin/flow-components/pull/3080
We should probably add a usage of these methods in a single selection IT as well...